rc/lsp.kak

```diff
@@ -1828,6 +1851,7 @@
     add-highlighter "%arg{1}/lsp_references" ranges lsp_references
     add-highlighter "%arg{1}/lsp_semantic_tokens_ranges" ranges lsp_semantic_tokens_ranges
     add-highlighter "%arg{1}/lsp_snippets_placeholders" ranges lsp_snippets_placeholders
+    add-highlighter "%arg{1}/lsp_document_links" ranges lsp_document_links
```

Comment:
should also add the corresponding `remove-highlighter` in `lsp-disable-impl`, for consistency